modules/python/except/source/generated/coda_except.py

```diff
@@ -104,12 +104,9 @@ class Context(_object):
     __getattr__ = lambda self, name: _swig_getattr(self, Context, name)
     __repr__ = _swig_repr
 
-    def __init__(self, *args):
-        """
-        __init__(except::Context self, std::string const & file, int line, std::string const & func, std::string const & time, std::string const & message) -> Context
-        __init__(except::Context self, Context c) -> Context
-        """
-        this = _coda_except.new_Context(*args)
+    def __init__(self, file, line, func, time, message):
```

Comment:
Why'd all the new Swig stuff come in..?

Reply:
Looks like it never got updated for https://github.com/mdaus/coda-oss/commit/6adcddf4013c4e383c5c0293507e7e4bc3773035

Reply:
Would it be worthwhile to make Jenkins complain if it generates SWIG code different than what's checked in?

Reply:
If you can figure out an easy way to do that, that would be awesome.  I guess it's just checking if 'git diff' finds anything?